.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Do not commit this file in the repository

Reply:
ok